README.md

```diff
@@ -110,6 +110,21 @@ Like node.js, callbacks for methods that accept them are optional but suggested
 you omit the callback, errors will be thrown as exceptions). The first callback parameter is
 reserved for passing errors. It will be `null` if no errors occurred and should always be checked.
 
+#### Support for Promises is here!
+Promise based API mimics the way Node [implements](https://nodejs.org/api/fs.html#fs_fs_promises_api) them. Both `Shell` and `FileSystem` now have `promises` object attached alongside regular callback style methods. Method names are identical to their callback counterparts with the difference that instead of receiving third argument as a callback, they return a Promise that is resolved or rejected based on the success of method execution.
```

Comment:
A few things to fix in this paragraph:

* "Promise based API" should be "The Promise based API"
* "now have `promises` object" should be "now have a `promises` object"
* "alongside regular callback style methods" should be "alongside the regular callback style methods"
* "instead of receiving third argument as a callback" should be "instead of taking a final callback argument"

Otherwise, looks great.